sympy/parsing/autolev/_listener_autolev_antlr.py

```diff
@@ -4,16 +4,17 @@
 
```

Comment:
Aren't these files autogenerated?

Reply:
This particular file doesn't look autogenerated. I can revert the change here though.

Reply:
I could be wrong about that.